scripts/wasm-analyze.py

```
@@ -0,0 +1,337 @@
+        print("Building release WASM...")
+        os.chdir(project_root)
+        result = subprocess.run(['zig', 'build', 'wasm'], capture_output=True, text=True)
+        if result.returncode != 0:
+            print(f"Error building WASM: {result.stderr}")
+            sys.exit(1)
+        # Debug build step may not exist, so we'll skip if it fails
+        if subprocess.run(['zig', 'build', '--help'], capture_output=True, text=True).stdout.find('wasm-debug') != -1:
+            print("Building debug WASM...")
+            result = subprocess.run(['zig', 'build', 'wasm-debug'], capture_output=True, text=True)
+            if result.returncode != 0:
+                print(f"Note: Debug build not available")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)